behat/tests/behat.yml

```diff
@@ -73,6 +73,12 @@ default:
             contexts:
                 - DigidepsBehat\Registration\RegistrationFeatureContext
 
+        userManagement:
+            description: Coverage of user management features
+            paths: [ "%paths.base%/features/userManagement" ]
+            contexts:
+                - DigidepsBehat\FeatureContext
```

Comment:
For the "new" structure that is domain/feature oriented, we have created a dir and namespace for each domain, e.g. `DigidepsBehat\UserManagement\UserManagementFeatureContext`, and in there is a FeatureContext file and a `UserManagementTrait` file. Would be good to adopt that here :)

Reply:
Yep makes sense -  I've added that now 👍